router/dnsmasq.go

```diff
@@ -30,14 +30,15 @@ func readAndParseLeases(filename string) (map[string]host, error) {
 		fmt.Printf("error opening file: %v\n", err)
 		return nil, err
 	}
-	balls := bufio.NewScanner(q)
-	for balls.Scan() {
-		host, err := parseMasq(balls.Text())
+	leasefile := bufio.NewScanner(q)
+	for leasefile.Scan() {
+		host, err := parseMasq(leasefile.Text())
 		if err != nil {
-			fmt.Printf("error parsing leaselines: %v\n", balls.Text())
+			fmt.Printf("error parsing leaselines: %v\n", leasefile.Text())
```

Comment:
Okay, so this commit belongs with the previous PR. Why did you choose to put it in this PR instead?